Error on Assertions.Terminal

Getting these errors:

```
#github.com/gdt-dev/kube
../../../../go/pkg/mod/github.com/gdt-dev/kube@v1.1.2-0.20230816130031-fbe369433eba/assertions.go:456:20: ja.Terminal undefined (type "github.com/gdt-dev/gdt/types".Assertions has no field or method Terminal)
../../../../go/pkg/mod/github.com/gdt-dev/kube@v1.1.2-0.20230816130031-fbe369433eba/eval.go:120:13: a.Terminal undefined (type "github.com/gdt-dev/gdt/types".Assertions has no field or method Terminal)
FAIL    functional-gdt [build failed]
FAIL
```

I am currently using:
	github.com/gdt-dev/gdt v1.2.2-0.20230815150028-c1bed53f824d
	github.com/gdt-dev/kube v1.1.2-0.20230816130031-fbe369433eba
	
It looks like [this commit](https://github.com/gdt-dev/gdt/commit/c1bed53f824da3113b8f24bf58a4eed2bcdf8733) in the gdt package with the removal of the Terminal interface has broken the Assertions in the gdt-dev/kube pacakge:

--- a/assertions.go
+++ b/assertions.go
@@ -453,7 +453,6 @@ func (a *assertions) jsonOK() bool {
 		}
 		ja := gdtjson.New(exp.JSON, b)
 		if !ja.OK() {
-			a.terminal = ja.Terminal()
 			for _, f := range ja.Failures() {
 				a.Fail(f)
 			}

--- a/eval.go
+++ b/eval.go
@@ -117,12 +117,11 @@ func (s *Spec) get(
 			a = s.doGet(ctx, t, c, res, name, s.Namespace())
 		}
 		success := a.OK()
-		term := a.Terminal()
 		debug.Println(
-			ctx, t, "%s (try %d after %s) ok: %v, terminal: %v",
-			s.Title(), attempts, after, success, term,
+			ctx, t, "%s (try %d after %s) ok: %v",
+			s.Title(), attempts, after, success,
 		)
-		if success || term {
+		if success {
 			ticker.Stop()
 			break
 		}

--- a/go.mod
+++ b/go.mod
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/cenkalti/backoff/v4 v4.2.1
-	github.com/gdt-dev/gdt v1.2.1
+	github.com/gdt-dev/gdt v1.3.0
 	github.com/samber/lo v1.38.1
 	github.com/stretchr/testify v1.8.4
 	gopkg.in/yaml.v3 v3.0.1

